### PR TITLE
[Gazebo] Adding gazebo v9

### PR DESCRIPTION
**Pull request by ruffsl**
And adding base image codenames tags

### CODE REVIEW

**Comment by ruffsl** [comment]
Hmm, not sure what format in the manifest has changed to fail the CI.

File: library/gazebo [hunk 1]
```diff
@@ -75,12 +75,30 @@ Directory: gazebo/7/ubuntu/xenial/libgazebo7
...
+
+Tags: gzserver9, gzserver9-xenial
+Architectures: amd64
+GitCommit: b8deb76898b59e0d43518f2cbdca6c90afc1debe
+Directory: gazebo/9/ubuntu/xenial/gzserver8
+
+Tags: libgazebo9, libgazebo9-xenial, latest
+Architectures: amd64
+GitCommit: b8deb76898b59e0d43518f2cbdca6c90afc1debe
+Directory: gazebo/9/ubuntu/xenial/libgazebo8
```
Comment (yosifkit) [inline library/gazebo]: Looks like it is because these new folders are still "8" (and so it can't scrape a `FROM` out of the Dockerfile).

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 215605a..8dd60d6 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,10 +1,12 @@
-gazebo:gzserver4 @ amd64
-gazebo:gzserver5 @ amd64
-gazebo:gzserver6 @ amd64
-gazebo:gzserver7 @ amd64
-gazebo:gzserver8 @ amd64
+gazebo:gzserver4-trusty @ amd64
+gazebo:gzserver5-trusty @ amd64
+gazebo:gzserver6-trusty @ amd64
+gazebo:gzserver7-xenial @ amd64
+gazebo:gzserver8-xenial @ amd64
+gazebo:gzserver9-xenial @ amd64
 gazebo:latest @ amd64
-gazebo:libgazebo4 @ amd64
-gazebo:libgazebo5 @ amd64
-gazebo:libgazebo6 @ amd64
-gazebo:libgazebo7 @ amd64
+gazebo:libgazebo4-trusty @ amd64
+gazebo:libgazebo5-trusty @ amd64
+gazebo:libgazebo6-trusty @ amd64
+gazebo:libgazebo7-xenial @ amd64
+gazebo:libgazebo8-xenial @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index f460612..ffe930d 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -1,11 +1,25 @@
 gazebo:gzserver4
+gazebo:gzserver4-trusty
 gazebo:gzserver5
+gazebo:gzserver5-trusty
 gazebo:gzserver6
+gazebo:gzserver6-trusty
 gazebo:gzserver7
+gazebo:gzserver7-xenial
 gazebo:gzserver8
+gazebo:gzserver8-xenial
+gazebo:gzserver9
+gazebo:gzserver9-xenial
 gazebo:latest
 gazebo:libgazebo4
+gazebo:libgazebo4-trusty
 gazebo:libgazebo5
+gazebo:libgazebo5-trusty
 gazebo:libgazebo6
+gazebo:libgazebo6-trusty
 gazebo:libgazebo7
+gazebo:libgazebo7-xenial
 gazebo:libgazebo8
+gazebo:libgazebo8-xenial
+gazebo:libgazebo9
+gazebo:libgazebo9-xenial
diff --git a/gazebo_gzserver4/Dockerfile b/gazebo_gzserver4-trusty/Dockerfile
similarity index 100%
rename from gazebo_gzserver4/Dockerfile
rename to gazebo_gzserver4-trusty/Dockerfile
diff --git a/gazebo_gzserver4/gzserver_entrypoint.sh b/gazebo_gzserver4-trusty/gzserver_entrypoint.sh
similarity index 100%
copy from gazebo_gzserver4/gzserver_entrypoint.sh
copy to gazebo_gzserver4-trusty/gzserver_entrypoint.sh
diff --git a/gazebo_gzserver5/Dockerfile b/gazebo_gzserver5-trusty/Dockerfile
similarity index 100%
rename from gazebo_gzserver5/Dockerfile
rename to gazebo_gzserver5-trusty/Dockerfile
diff --git a/gazebo_gzserver5/gzserver_entrypoint.sh b/gazebo_gzserver5-trusty/gzserver_entrypoint.sh
similarity index 100%
rename from gazebo_gzserver5/gzserver_entrypoint.sh
rename to gazebo_gzserver5-trusty/gzserver_entrypoint.sh
diff --git a/gazebo_gzserver6/Dockerfile b/gazebo_gzserver6-trusty/Dockerfile
similarity index 100%
rename from gazebo_gzserver6/Dockerfile
rename to gazebo_gzserver6-trusty/Dockerfile
diff --git a/gazebo_gzserver6/gzserver_entrypoint.sh b/gazebo_gzserver6-trusty/gzserver_entrypoint.sh
similarity index 100%
rename from gazebo_gzserver6/gzserver_entrypoint.sh
rename to gazebo_gzserver6-trusty/gzserver_entrypoint.sh
diff --git a/gazebo_gzserver7/Dockerfile b/gazebo_gzserver7-xenial/Dockerfile
similarity index 100%
rename from gazebo_gzserver7/Dockerfile
rename to gazebo_gzserver7-xenial/Dockerfile
diff --git a/gazebo_gzserver7/gzserver_entrypoint.sh b/gazebo_gzserver7-xenial/gzserver_entrypoint.sh
similarity index 100%
rename from gazebo_gzserver7/gzserver_entrypoint.sh
rename to gazebo_gzserver7-xenial/gzserver_entrypoint.sh
diff --git a/gazebo_gzserver8/Dockerfile b/gazebo_gzserver8-xenial/Dockerfile
similarity index 100%
copy from gazebo_gzserver8/Dockerfile
copy to gazebo_gzserver8-xenial/Dockerfile
diff --git a/gazebo_gzserver8/gzserver_entrypoint.sh b/gazebo_gzserver8-xenial/gzserver_entrypoint.sh
similarity index 100%
rename from gazebo_gzserver8/gzserver_entrypoint.sh
rename to gazebo_gzserver8-xenial/gzserver_entrypoint.sh
diff --git a/gazebo_gzserver8/Dockerfile b/gazebo_gzserver9-xenial/Dockerfile
similarity index 89%
rename from gazebo_gzserver8/Dockerfile
rename to gazebo_gzserver9-xenial/Dockerfile
index d580595..c68a6e0 100644
--- a/gazebo_gzserver8/Dockerfile
+++ b/gazebo_gzserver9-xenial/Dockerfile
@@ -1,4 +1,4 @@
-# This is an auto generated Dockerfile for gazebo:gzserver8
+# This is an auto generated Dockerfile for gazebo:gzserver9
 # generated from docker_images/create_gzserver_image.Dockerfile.em
 FROM ubuntu:xenial
 
@@ -12,7 +12,7 @@ RUN . /etc/os-release \
 
 # install gazebo packages
 RUN apt-get update && apt-get install -q -y \
-    gazebo8=8.2.0-1* \
+    gazebo9=9.0.0-1* \
     && rm -rf /var/lib/apt/lists/*
 
 # setup environment
diff --git a/gazebo_gzserver4/gzserver_entrypoint.sh b/gazebo_gzserver9-xenial/gzserver_entrypoint.sh
similarity index 100%
rename from gazebo_gzserver4/gzserver_entrypoint.sh
rename to gazebo_gzserver9-xenial/gzserver_entrypoint.sh
diff --git a/gazebo_latest/Dockerfile b/gazebo_latest/Dockerfile
index a1a54ad..c5b0279 100644
--- a/gazebo_latest/Dockerfile
+++ b/gazebo_latest/Dockerfile
@@ -1,7 +1,7 @@
-# This is an auto generated Dockerfile for gazebo:libgazebo8
+# This is an auto generated Dockerfile for gazebo:libgazebo9
 # generated from docker_images/create_gzclient_image.Dockerfile.em
-FROM gazebo:gzserver8
+FROM gazebo:gzserver9
 # install gazebo packages
 RUN apt-get update && apt-get install -q -y \
-    libgazebo8-dev=8.2.0-1* \
+    libgazebo9-dev=9.0.0-1* \
     && rm -rf /var/lib/apt/lists/*
diff --git a/gazebo_libgazebo4/Dockerfile b/gazebo_libgazebo4-trusty/Dockerfile
similarity index 100%
rename from gazebo_libgazebo4/Dockerfile
rename to gazebo_libgazebo4-trusty/Dockerfile
diff --git a/gazebo_libgazebo5/Dockerfile b/gazebo_libgazebo5-trusty/Dockerfile
similarity index 100%
rename from gazebo_libgazebo5/Dockerfile
rename to gazebo_libgazebo5-trusty/Dockerfile
diff --git a/gazebo_libgazebo6/Dockerfile b/gazebo_libgazebo6-trusty/Dockerfile
similarity index 100%
rename from gazebo_libgazebo6/Dockerfile
rename to gazebo_libgazebo6-trusty/Dockerfile
diff --git a/gazebo_libgazebo7/Dockerfile b/gazebo_libgazebo7-xenial/Dockerfile
similarity index 100%
rename from gazebo_libgazebo7/Dockerfile
rename to gazebo_libgazebo7-xenial/Dockerfile
diff --git a/gazebo_latest/Dockerfile b/gazebo_libgazebo8-xenial/Dockerfile
similarity index 100%
copy from gazebo_latest/Dockerfile
copy to gazebo_libgazebo8-xenial/Dockerfile
```

</details>

**Comment by yosifkit** [comment]
Build test of #3968; 9a25783b; `amd64` (`gazebo`):

```console

$ bashbrew build gazebo:gzserver4
Building bashbrew/cache:bd503a9f844d7ace7736a9fc4723b642edc95373536cf389b253306c2a4fdab5 (gazebo:gzserver4)
Tagging gazebo:gzserver4
Tagging gazebo:gzserver4-trusty

$ test/run.sh gazebo:gzserver4
testing gazebo:gzserver4
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gazebo:libgazebo4
Building bashbrew/cache:9e65978a7a954b6abf354b3f6abd822540cc5d4239f1f19a3febee4bb6b86505 (gazebo:libgazebo4)
Tagging gazebo:libgazebo4
Tagging gazebo:libgazebo4-trusty

$ test/run.sh gazebo:libgazebo4
testing gazebo:libgazebo4
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gazebo:gzserver5
Building bashbrew/cache:2c170ef3f57364ea123937ee63f200c2a881b8d2126ac1736de45e78475f428b (gazebo:gzserver5)
Tagging gazebo:gzserver5
Tagging gazebo:gzserver5-trusty

$ test/run.sh gazebo:gzserver5
testing gazebo:gzserver5
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gazebo:libgazebo5
Building bashbrew/cache:6acc1fffbdcae337af013faf1a7709d06ea038827cae553b5d36b222624ac76d (gazebo:libgazebo5)
Tagging gazebo:libgazebo5
Tagging gazebo:libgazebo5-trusty

$ test/run.sh gazebo:libgazebo5
testing gazebo:libgazebo5
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gazebo:gzserver6
Building bashbrew/cache:645ad6bbef73197605e8228298dbb025e806949bcbc476f60b014d0f119e5a40 (gazebo:gzserver6)
Tagging gazebo:gzserver6
Tagging gazebo:gzserver6-trusty

$ test/run.sh gazebo:gzserver6
testing gazebo:gzserver6
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gazebo:libgazebo6
Building bashbrew/cache:bccbabf3828f1f52175de2d818d878749cb09e696382f2fcd3876a7b591322f9 (gazebo:libgazebo6)
Tagging gazebo:libgazebo6
Tagging gazebo:libgazebo6-trusty

$ test/run.sh gazebo:libgazebo6
testing gazebo:libgazebo6
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gazebo:gzserver7
Building bashbrew/cache:1e55fa962656ec144c3f5d178049a362c1c416056bd3d984eedc390c7dc411b6 (gazebo:gzserver7)
Tagging gazebo:gzserver7
Tagging gazebo:gzserver7-xenial

$ test/run.sh gazebo:gzserver7
testing gazebo:gzserver7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gazebo:libgazebo7
Building bashbrew/cache:99ae14d54c925ef9ef9c2b0a9e6a79a013ed98c90bd8b8d51858cb6df28ed79c (gazebo:libgazebo7)
Tagging gazebo:libgazebo7
Tagging gazebo:libgazebo7-xenial

$ test/run.sh gazebo:libgazebo7
testing gazebo:libgazebo7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gazebo:gzserver8
Building bashbrew/cache:dcfbe67a04fdfbc44eb689b4b6cd7301e72713da802fc2453dd4ce8644947228 (gazebo:gzserver8)
Tagging gazebo:gzserver8
Tagging gazebo:gzserver8-xenial

$ test/run.sh gazebo:gzserver8
testing gazebo:gzserver8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gazebo:libgazebo8
Building bashbrew/cache:332c17c3469642213c380925e0e1f76ffcc2030ca531f8077321587b58a61d60 (gazebo:libgazebo8)
Tagging gazebo:libgazebo8
Tagging gazebo:libgazebo8-xenial

$ test/run.sh gazebo:libgazebo8
testing gazebo:libgazebo8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gazebo:gzserver9
Building bashbrew/cache:acac70998fa0b3f17e2f69e40e757c7f4c8dfc4bd510916bfd9bd658a8f0f76e (gazebo:gzserver9)
Tagging gazebo:gzserver9
Tagging gazebo:gzserver9-xenial

$ test/run.sh gazebo:gzserver9
testing gazebo:gzserver9
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gazebo:libgazebo9
Building bashbrew/cache:08176fbba474991b3884e100d6a337fba90aff3b9ad98c5744b24171b358380a (gazebo:libgazebo9)
Tagging gazebo:libgazebo9
Tagging gazebo:libgazebo9-xenial
Tagging gazebo:latest

$ test/run.sh gazebo:libgazebo9
testing gazebo:libgazebo9
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**